gateway/api/views.py

```diff
@@ -114,16 +116,63 @@ def get_model_serializer_run_program(*args, **kwargs):
     def get_serializer_class(self):
         return self.serializer_class
 
+    def get_object(self):
+        logger.warning("ProgramViewSet.get_object not implemented")
+
     def get_queryset(self):
-        # Allow unauthenticated users to read the swagger documentation
```

Comment:
will swagger access (whenever we get that actually working) now require authentication?

Reply:
this is something I'm not sure, I was testing this locally and for me was working fine so I removed it but if you can double check perfect

(spoiler: you can enter in swagger but to run it correctly you need to add Authorization header authenticating in swager, what I think it has sense from my point of view)